### ISSUE TITLE
Specify OccluderPolygon2d from CollisionShape2d

### ISSUE THREAD

**Issue by BluePhoenixGame** [godotengine/godot-proposals#564]
**Describe the project you are working on:**
A simple arcade game. It uses light and collisionshapes.

**Describe the problem or limitation you are having in your project:**
Currently, it's hard to make a circle or other polygon with a OccluderPolygon2d

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
I think either a setting to use the collisionshape of the parent node as a lightoccluder, or you can in the drop-down let users specify an already existing collisionshape (sibling-nodes)

**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
So far it doesn't make sense to have any code or any API for it.
![image](https://user-images.githubusercontent.com/38372706/76128541-e8cb6180-6004-11ea-9fc9-7ae22c62f877.png)
In the red circle another option "From CollisionShape2d". There you can later specify a collision shape out of all the sibling nodes.

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
There is currently no way that this could be recreated in a simple manor.

**Is there a reason why this should be core and not an add-on in the asset library?:**
This could be such a broadly used feature. And very convenient for most cases when working with lightning.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/6750.

Note that CollisionShape2Ds may be more precise than their visual representation suggests. Therefore, their visual representation would have to be used as a base for the LightOccluder2D as it needs to have a finite amount of polygons.

**Comment by BluePhoenixGame** [comment]
Yes, the visual representation was what i had in mind and it would not be a problem since a LightOccluder2D doesn't need to be as precise as a CollisionShape2D needs to be.
